ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2542,3 +2542,85 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: app.ci
+  cron: 0 * */14 * *
```

Comment:
This looks like an update every hour on the 14th/28th of each month, or am I reading this wrong? Something like 0 0 */14 * * should do the trick. Actually how about every week for now so we can see how this flows.